packages/cli/templates/web-library-package/README.md.hbs

```diff
@@ -0,0 +1,12 @@
+# {{id}}
+
+_This package was created through the Backstage CLI_.
+
+## Installation
+
+Install the package via Yarn:
+
+```sh
+cd <package-dir> # if within a monorepo
+yarn add @backstage/{{id}}
```

Comment:
This might cause confusion. Should this be `yarn add {{name}}`?

Reply:
Nice catch!

Reply:
👍